Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 resolver = "2"
-members = ["lib", "primitives", "raiko-guest", "raiko-host"]
```

Comment:
Unfortunately, I had to manually add some precompiled libraries (`*.so`) to the `sgx-ra` directory. They are all third party libraries that need to be built in a carefully crafted environment (there is no Docker container available to make building easier).

Reply:
It's a question. Maybe we can add the building logic in the build.rs

Reply:
I'll leave it as it is for now because it's not that trivial. RA-TLS is not used in the current `raiko-host` implementation anyway; instead we use `one-shot` mode, i.e. `raiko-host` runs `raiko-guest` which terminates immediately after printing the proofs.